calculators/candidate_expenditures_by_type.rb

```diff
@@ -65,7 +65,21 @@ def expenditures_by_candidate_by_type
         ORDER BY "Expn_Code", "Filer_ID", "Report_Num" DESC
       SQL
 
-      results.each do |result|
+      # 497 does not contain "Expn_Code" making this calculator pretty useless
+      # for those contributions.
+      # To make the numbers line up closer, we'll bucket those all under "Not
+      # Stated".
+      late_expenditures = ActiveRecord::Base.connection.execute(<<-SQL)
+        SELECT DISTINCT ON ("Filer_ID")
+          "Filer_ID", '' AS "Expn_Code", SUM("Amount") AS "Total"
```

Comment:
Should `''` be `'Not stated'`? Or maybe `null` so it's easier to handle on the frontend?